### ISSUE TITLE
Error when installing

### ISSUE THREAD

**Issue by csbruno** [browniebroke/deezer-python#27]
`Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\b\AppData\Local\Temp\pip-install-osg8z63j\deezer-python\setup.py", line 20, in <module>
        read('AUTHORS.rst')),
      File "C:\Users\b\AppData\Local\Temp\pip-install-osg8z63j\deezer-python\setup.py", line 9, in read
        return f.read()
      File "c:\users\b\appdata\local\programs\python\python37\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 35: character maps to <undefined>`

**Comment by browniebroke** [comment]
It looks like you're using Python 3.7, which isn't well supported yet due to #26. In the meantime, I think we can add Python 3.7 to the build matrix under `allow_failures`:

- Appveyor: https://www.appveyor.com/docs/build-configuration/#allow-failing-jobs
- Travis: https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail

**Comment by browniebroke** [comment]
It all seems to work fine on Windows, but tests are breaking in Travis, looks like it hangs in the middle of one test.

Could you try to install from the git repo to confirm it's fixed? I'll make a new release shortly after.

**Comment by k33k00** [comment]
seems to be working fine from the repo with python 3.7

```
pip install https://github.com/browniebroke/deezer-python/archive/master.zip
Collecting https://github.com/browniebroke/deezer-python/archive/master.zip
  Downloading https://github.com/browniebroke/deezer-python/archive/master.zip
     \ 1.2MB 20.4MB/s
Collecting tornado (from deezer-python==0.7.0)
  Using cached https://files.pythonhosted.org/packages/d7/d5/99aff77c15da335e3dbf30254b304a526494dca8e51b4b47419772e35a4e/tornado-5.1.1-cp37-cp37m-win_amd64.whl
Collecting six (from deezer-python==0.7.0)
  Using cached https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting requests (from deezer-python==0.7.0)
  Using cached https://files.pythonhosted.org/packages/65/47/7e02164a2a3db50ed6d8a6ab1d6d60b69c4c3fdf57a284257925dfc12bda/requests-2.19.1-py2.py3-none-any.whl
Collecting chardet<3.1.0,>=3.0.2 (from requests->deezer-python==0.7.0)
  Using cached https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl
Collecting idna<2.8,>=2.5 (from requests->deezer-python==0.7.0)
  Using cached https://files.pythonhosted.org/packages/4b/2a/0276479a4b3caeb8a8c1af2f8e4355746a97fab05a372e4a2c6a6b876165/idna-2.7-py2.py3-none-any.whl
Collecting certifi>=2017.4.17 (from requests->deezer-python==0.7.0)
  Using cached https://files.pythonhosted.org/packages/df/f7/04fee6ac349e915b82171f8e23cee63644d83663b34c539f7a09aed18f9e/certifi-2018.8.24-py2.py3-none-any.whl
Collecting urllib3<1.24,>=1.21.1 (from requests->deezer-python==0.7.0)
  Using cached https://files.pythonhosted.org/packages/bd/c9/6fdd990019071a4a32a5e7cb78a1d92c53851ef4f56f62a3486e6a7d8ffb/urllib3-1.23-py2.py3-none-any.whl
Building wheels for collected packages: deezer-python
  Running setup.py bdist_wheel for deezer-python ... done
  Stored in directory: C:\Users\Kieran\AppData\Local\Temp\pip-ephem-wheel-cache-q2k3y0b1\wheels\a2\16\3d\773b49b4d06e2153726ab07abe31c87f592e7a47d32f62dff1
Successfully built deezer-python
Installing collected packages: tornado, six, chardet, idna, certifi, urllib3, requests, deezer-python
Successfully installed certifi-2018.8.24 chardet-3.0.4 deezer-python-0.7.0 idna-2.7 requests-2.19.1 six-1.11.0 tornado-5.1.1 urllib3-1.23
```

**Comment by browniebroke** [comment]
The latest PYPI release has the fix as well: 0.7.0
